debian/ags+libraries/hooks/B00_copy_libs.sh

```diff
@@ -17,23 +17,19 @@ BIT=32
   mkdir $BINDMOUNT/data/lib$BIT
 
   for library in \
-    liballeg.so.4.4 \
-    libaldmb.so.1 \
-    libdumb.so.1 \
     libogg.so.0 \
     libtheora.so.0 \
     libvorbis.so.0 \
-    libvorbisfile.so.3 \
-    allegro/4.4.2/alleg-alsadigi.so \
-    allegro/4.4.2/alleg-alsamidi.so \
-    allegro/4.4.2/modules.lst; do
+    libvorbisfile.so.3; do
       cp -L /usr/lib/$TRIPLET/$library $BINDMOUNT/data/lib$BIT
   done
-
+  
+  for library in \
+    libSDL2-2.0.so.0; do
+      cp -L /usr/local/lib/$library $BINDMOUNT/data/lib$BIT
```

Comment:
why would you have SDL2 installed in /usr/local and everything else in /usr ? looks like a configuration error when compiling/installing SDL2.

Reply:
Other libraries are from Debian Jessie repository. SDL2 is built from source.

Reply:
Also, no one should ship system SDL2 for others, SDL2 from package management comes configured to prefer linking directly instead of using dlopen. While this may work great in "my computer ™️" it's best to let SDL2 figure out what exists or not in the computer (alsa, pulseaudio, oss, framebuffer, ...).